src/app/globals.css

```diff
@@ -2,20 +2,13 @@
 @tailwind components;
 @tailwind utilities;
 
-:root {
-  --background: #ffffff;
-  --foreground: #171717;
-}
-
-@media (prefers-color-scheme: dark) {
-  :root {
-    --background: #0a0a0a;
-    --foreground: #ededed;
-  }
+@font-face {
+  font-family: 'Geist';
+  src: url(../../public/fonts/GeistMonoVF.woff) format('woff');
+  font-weight: normal;
+  font-style: normal;
 }
 
 body {
-  color: var(--foreground);
```

Comment:
Do we need to delete all these?

Reply:
yes sir because we are doing our own so if we dont change so it will affect to our styling we done in code

Reply:
@kunaldevxxx This is global stylesheet. Any change here impact every single page. Can we have Auth page specific styling within the Auth page itself or just create `auth.css` and include it in `Auth.tsx` to do Auth page specific styles please? This will ensure that Auth page specific styles are not coupled with other pages.